docs/conf.py

```diff
@@ -66,9 +66,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = "0.7.1"
+version = "0.8.0"
 # The full version, including alpha/beta/rc tags.
-release = "0.7.1"
+release = "0.8.0"
```

Comment:
We never update this in regular Mesa. Maybe we should start doing this. Good catch.